src/pages/overview/pdf-services-api/howtos/pdf-properties.md

```diff
@@ -34,38 +34,39 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
     public static void main(String[] args) {
 
       try {
-
-        // Initial setup, create credentials instance.
-        Credentials credentials = Credentials.servicePrincipalCredentialsBuilder()
-           .withClientId("PDF_SERVICES_CLIENT_ID")
-           .withClientSecret("PDF_SERVICES_CLIENT_SECRET")
-           .build();
-
-        //Create an ExecutionContext using credentials and create a new operation instance.
-        ExecutionContext executionContext = ExecutionContext.create(credentials);
-        PDFPropertiesOperation pdfPropertiesOperation = PDFPropertiesOperation.createNew();
-
-        // Provide an input FileRef for the operation
-        FileRef source = FileRef.createFromLocalFile("src/main/resources/pdfPropertiesInput.pdf");
-        pdfPropertiesOperation.setInputFile(source);
-
-        // Build PDF Properties options to include page level properties and set them into the operation
-        PDFPropertiesOptions pdfPropertiesOptions = PDFPropertiesOptions.PDFPropertiesOptionsBuilder()
-              .includePageLevelProperties(true)
-              .build();
-        pdfPropertiesOperation.setOptions(pdfPropertiesOptions);
-
-        // Execute the operation ang get properties of the PDF in PDFProperties object.
-        PDFProperties result = pdfPropertiesOperation.execute(executionContext);
-            
-        // Get properties of the PDF
-        LOGGER.info("The Page level properties of the PDF: {}", result.getDocument().getPageCount());
-        LOGGER.info("The Fonts used in the PDF: ");
-        for(Font font: result.getDocument().getFonts()) {
-            LOGGER.info(font.getName());
-        }
-
-      } catch (ServiceApiException | IOException | SdkException | ServiceUsageException ex) {
+        // Initial setup, create credentials instance
+        Credentials credentials = new ServicePrincipalCredentials(
+                System.getenv("PDF_SERVICES_CLIENT_ID"),
+                System.getenv("PDF_SERVICES_CLIENT_SECRET"));
+
+        // Creates a PDF Services instance
+        PDFServices pdfServices = new PDFServices(credentials);
+
+        // Creates an asset from source file and upload
+        InputStream inputStream = Files.newInputStream(new File("src/main/resources/pdfPropertiesInput.pdf").toPath());
+        Asset asset = pdfServices.upload(inputStream, PDFServicesMediaType.PDF.getMediaType());
+
+        // Create parameters for the job
```

Comment:
same

Reply:
As discussed, IG this is from already reviewed samples.